DavidGonzalezZumaya/Day3/ChallengeDay3/PruebaFigura.java

```diff
@@ -0,0 +1,22 @@
+public class PruebaFigura {
+    public static void main(String[] args) {
+
+        Figura[] figuras = new Figura[3]; // arreglo de referencias a figuras
+
+        figuras[0] = new Triangulo("morado", 4.3, 8.6);
+        figuras[1] = new Circulo("turquesa", 2.5);
+        figuras[2] = new Rectangulo("rosa", 15.2, 7.5);
+
+        for (int i = 0; i < 3; i++) {
```

Comment:
This is redundant. If you are defiying the array size as 3. just use figuras.length
Use meaningful var names **"i"**